presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,30 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that runs the Presto process. These limits 
+  may depend on the specific Linux distribution you are using. Currently,
+  Presto requires a minimum of 4096 open file descriptors, and strongly
+  recommends at least 8192 when it runs. However, the number of open file
```

Comment:
We can remove the "when it runs". I also think it's strange to say "Presto recommends" since it's software. *we* are the ones recommending.

Do we need to mention both 4096 and 8192? Seems better to just give a single recommended minimum to avoid confusion.